src/main/java/io/leonis/subra/game/data/MovingPlayer.java

```diff
@@ -46,15 +45,25 @@ default INDArray getXYVelocity() {
     return this.getState().getMean().get(NDArrayIndex.interval(4, 6), NDArrayIndex.all());
   }
 
+  /**
+   * {@inheritDoc}
```

Comment:
Isn't this an implicit default?

Reply:
it is, but codacy complained about it before so i've added it